app/routes/auth/auth.tsx

```diff
@@ -0,0 +1,9 @@
+import type { ActionArgs } from "@remix-run/server-runtime"
+import { redirect } from "@remix-run/server-runtime"
+import { authenticator } from "~/services/auth/auth.server"
+
+export let loader = () => redirect("/admin")
+
+export let action = ({ request }: ActionArgs) => {
+  return authenticator().authenticate("auth0", request)
+}
```

Comment:
What's the point of this code by the way? It's not being used by anything else right now but you've exported the functions so I'm assuming you're intending to use it eventually